dask_sql/context.py

```diff
@@ -270,12 +267,6 @@ def create_table(
         self.schema[schema_name].tables[table_name.lower()] = dc
         self.schema[schema_name].statistics[table_name.lower()] = statistics
 
-        # Register the table with the Rust DaskSQLContext
-        self.context.register_table(
```

Comment:
We were registering the table here without any columns, and repeating the whole thing in `_prepare_schemas` again with columns.

Reply:
Good catch!